ergo/conditions/histogram.py

```diff
@@ -27,18 +27,22 @@ def loss(self, dist):
         return self.weight * total_loss / self.xs.size
 
     def normalize(self, scale: Scale):
-        normalized_xs = np.array([scale.normalize_point(x) for x in self.xs])
-        normalized_densities = np.array(
-            [density * scale.width for density in self.densities]
-        )
+        # TODO: Should we really just returns self here? Because the pdf and returned density are
```

Comment:
Functionally I think this is fine. Convention-wise, it also seems consistent. Is there anything in the way of dropping the scale argument in normalization in all conditions? Then it would be consistent with the convention in `distributions/` where normalize does not take a scale argument.